pkgs/applications/misc/calibre/default.nix

```diff
@@ -51,6 +90,11 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://raw.githubusercontent.com/debian-calibre/calibre/debian/${finalAttrs.version}+ds-1/debian/patches/hardening/0007-Hardening-Qt-code.patch";
       hash = "sha256-a6yyG0RUsQJBBNxeJsTtQSBV2lxdzz1hnTob88O+SKg=";
     })
+    (fetchpatch {
+      name = "build-from-git-without-internet";
+      url = "https://github.com/kovidgoyal/calibre/commit/4b7c79f71821e9d13edff5d04249c5decf1093a0.patch";
```

Comment:
Since this is unofficial, I believe it would be easier for people to maintain the patch by putting it in nixpkgs directory. For example the change of variable names `CACERT -> CALIBRE_BOOTSTRAP_CACERT` can be more easily accomplished this way. What do you think?

Reply:
You are right. Done.